src/Dog.java

```diff
@@ -0,0 +1,8 @@
+public class Dog extends Mammal {
```

Comment:
this class should have a constructor with a dogVariable, mammal and animalVariable as parameters and call super(mammalVariable, animalVariable) in the first line)